projects/framework-blender/pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[tool.poetry]
+name = "ftrack-framework-blender"
+version = "24.6.0"
+description='ftrack Blender integration'
+authors = ["ftrack Integrations Team <integrations@backlight.co>"]
+readme = "README.md"
+packages = [{include = "ftrack_framework_blender", from = "source"}]
+license = "Apache-2.0"
+homepage = "https://ftrack.com"
+repository = "https://github.com/ftrackhq/integrations/tree/main/projects/framework-blender"
+
+[[tool.poetry.source]]
+name = "PyPI"
+priority = "primary"
+
+[[tool.poetry.source]]
+name = "testpypi"
+url = "https://test.pypi.org/simple/"
+priority = "explicit"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry.dependencies]
+python = ">=3.7, <3.12"
```

Comment:
Python 3.11 is the only compatible one, right? no previous versions.

Reply:
seems the case, fixing this.